rotkehlchen/tests/unit/test_accounting.py

```diff
@@ -541,7 +542,8 @@ def test_settlement_buy(accountant):
         end_ts=1519693374,
         history_list=history,
     )
-    assert accountant.events.cost_basis.get_calculated_asset_amount('BTC').is_close('3.9908725')
+    # asset amount here is 3.9908725 + 0.0012 (from the fee)
```

Comment:
How can that be possible? The fee is something you pay. If anything the `0.0012` should be deducted from somewhere.

Reply:
Yep. My fault, there is something still off. Going to check it